dataprep/eda/diff/compute/multiple_df.py

```diff
@@ -0,0 +1,368 @@
+        Get the specified index for all elements in the list.
+        """
+        output = []
+        for data in self.data:
+            output.append(data[ind])
+
+        return output
+
+    def getmask(
+        self, mask: Union[List[dd.Series], UserList], inverse: bool = False
+    ) -> List[dd.Series]:
+        """
+        Return rows based on a boolean mask.
+        """
+        output = []
+        for data, cond in zip(self.data, mask):
+            if inverse:
+                output.append(data[~cond])
+            else:
+                output.append(data[cond])
+
+        return output
+
+    def self_map(self, func: Callable[[dd.Series], Any], **kwargs: Any) -> List[Any]:
+        """
+        Map the data to the given function.
+        """
+        return [func(srs, **kwargs) for srs in self.data]
+
+
+def compare_multiple_df(
+    df_list: List[dd.DataFrame], cfg: Config, dtype: Optional[DTypeDef]
+) -> Intermediate:
+    """
+    Compute function for plot_diff([df...])
+
+    Parameters
+    ----------
+    dfs
+        Dataframe sequence to be compared.
+    cfg
+        Config instance
+    dtype: str or DType or dict of str or dict of DType, default None
+        Specify Data Types for designated column or all columns.
+        E.g.  dtype = {"a": Continuous, "b": "Nominal"} or
+        dtype = {"a": Continuous(), "b": "nominal"}
+        or dtype = Continuous() or dtype = "Continuous" or dtype = Continuous()
+    """
+    # pylint: disable=too-many-branches, too-many-locals
+
+    dfs = Dfs(df_list)
+    dfs_cols = dfs.columns.apply("to_list")
+
+    labeled_cols = dict(zip(cfg.diff.label, dfs_cols))
+    baseline: int = cfg.diff.baseline
+    data: List[Any] = []
+    aligned_dfs = dd.concat(df_list, axis=1)
+
+    if len(set.intersection(*map(set, dfs_cols))) < 3:
+        raise DataprepError("The input doesn't have enough resemblance.")
+
+    # OrderedDict for keeping the order
+    uniq_cols = list(OrderedDict.fromkeys(sum(dfs_cols, [])))
+
+    for col in uniq_cols:
+        srs = Srs(aligned_dfs[col])
+        col_dtype = srs.self_map(detect_dtype, known_dtype=dtype)
+        if len(col_dtype) > 1:
+            col_dtype = col_dtype[baseline]
+        else:
+            col_dtype = col_dtype[0]
+
+        orig = [src for src, seq in labeled_cols.items() if col in seq]
+
+        if is_dtype(col_dtype, Continuous()) and cfg.hist.enable:
+            data.append((col, Continuous(), _cont_calcs(srs.apply("dropna"), cfg), orig))
+        elif is_dtype(col_dtype, Nominal()) and cfg.bar.enable:
+            srs = srs.apply("astype(str)")
+            data.append((col, Nominal(), _nom_calcs(srs.apply("dropna"), cfg), orig))
+        elif is_dtype(col_dtype, DateTime()) and cfg.line.enable:
+            data.append(
+                (col, DateTime(), dask.delayed(_calc_line_dt)(srs, col, cfg.line.unit), orig)
+            )
+
+    stats = calc_stats(dfs, cfg, dtype)
+    data = [dask.compute(*i) for i in data]
```

Comment:
Do you think it's possible to use fewer calls of `dask.compute()` throughout this PR? From my understanding [`dask.compute`](https://app.circleci.com/pipelines/github/sfu-db/dataprep/2303/workflows/0490c94c-d7bc-4e51-9bfd-fa7e2d551b0a/jobs/5999) will traverse any python collections such as this list and compute the dask objects. If only one call to `dask.compute` is made then the operations can be optimized.

Reply:
That would be ideal, however it doesn't work well on `stats` because some computations return a `dd.Scalar` object which is not a delayed object.

The data will be changed to call compute only once.

Reply:
I'm not sure I understand, so `dask.compute()` will not compute a `dd.Scalar` object?

Reply:
At least it won't compute something like this: `{..., 'key': [dd.Scalar....], ...}`

Reply:
It looks like you're talking about something like this

![Screen Shot 2021-03-31 at 3 30 01 PM](https://user-images.githubusercontent.com/25874021/113219339-0aae0c80-9236-11eb-9784-d7d0b0228d1b.png)

Reply:
Well this is interesting. Maybe it doesn't work when the dictionary contains `dask.delayed` object, this is what in this stats exactly.

This is what stats looks like before calling compute:
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/12697964/113220204-a12efd80-9237-11eb-9215-bb420732e5f8.png">

After:
<img width="1044" alt="image" src="https://user-images.githubusercontent.com/12697964/113220214-a8eea200-9237-11eb-9bb0-4efb65d9a816.png">

Reply:
Ya this is interesting, I'm not sure why that's happening. I cant' seem to reproduce the problem with a trivial example. 
![Screen Shot 2021-03-31 at 4 25 33 PM](https://user-images.githubusercontent.com/25874021/113223417-f968fe00-923d-11eb-8143-a8dc9c1bb787.png)

Reply:
Looks like there is something wrong with `Dfs` and `Srs` object.

Reply:
I figured out this part, will make another commit.